frontend/components/FieldMedia/FieldMediaEdit.jsx

```diff
@@ -124,8 +125,7 @@ export default class FieldImageEdit extends Component {
       value
     } = this.props
 
-    const fieldImage = config.FieldImage || {}
-    const accept = fieldImage.accept
+    const accept = schema.validation && schema.validation.accept || ['*/*']
```

Comment:
Is `accept` the right name for this validation operator? Wouldn't `validation.mimeType` be more descriptive?